modules/getting-started/proc-customize-rhdh-learning-paths.adoc

```diff
@@ -19,7 +19,9 @@ You can provide data to the Learning Path from the following sources:
 
 .Prerequisites
 
-You have installed {product} by using either the Operator or Helm chart. For more information, see xref:proc-install-rhdh_rhdh-getting-started[].
+You have installed {product} by using either the Operator or Helm chart.
+//For more information, see xref:proc-install-rhdh_rhdh-getting-started[].
+//replace with link to installation guide.
```

Comment:
Are we going to provide this link later??
Also applicable to other instances below

Reply:
Good Q - We are going to do this globally under a separate PR